src/pages/iou/IOUModal.js

```diff
@@ -138,6 +179,33 @@ class IOUModal extends Component {
         this.setState({selectedCurrency});
     }
 
+    closeModal() {
+        Navigation.dismissModal();
+    }
+
+    createTransaction({debtorEmail, splits, participants}) {
+        if (debtorEmail) {
+            return createIOUTransaction({
+                comment: this.state.comment,
+
+                // should send in cents to API
+                amount: this.state.amount * 100,
```

Comment:
Hmm, now I'm not so sure this will work. What about the other currencies? 😕 

Maybe the API should take currency and amount and work it out.

Reply:
Yeah, It felt odd while I was doing this too. Not sure what to do with this one.

Reply:
The currency is passed to the API command. I don't see any problem here, what's the issue?

Reply:
Yeah, it's front-ends problem but we'll solve it in a separate issue. I think we'll need to work out the decimal place for each currency [like we do for Web-E](https://github.com/Expensify/Web-Expensify/blob/40b65b3795fef502632ed1d2d7a537583bbf5738/site/lib/forex.jsx#L135)

Reply:
Oh ok. I we might be able to use `Intl` for this (which we are adding as part of the internationalization project) ie:
```
Intl.NumberFormat('en-US',{
    style: "currency",
    currency: "JPY"
}).format(5.15)
"¥5"
```